### ISSUE TITLE
[Nodes] Sub Nodes icon are not working

### ISSUE THREAD

**Issue by Nirjonadda** [Audentio/xf2addon-issues#2103]
@LukasWieditz from previous bug fix, now no more icon show for unread mark and read mark wrong icon show in forum list but forum view node icon showing correctly.

See here: https://gfycat.com/scaredjovialfantail

**Comment by LukasWieditz** [comment]
I'm not able to reproduce that anywhere. Which style are you using?

**Comment by Nirjonadda** [comment]
UIX Pro style.

**Comment by LukasWieditz** [comment]
That doesn't look like the UI.X Pro Style to me. 🤔 Anyhow, it's working there too for me. Did you make any custom changes that may have an influence on that?

![yKFi6dp](https://user-images.githubusercontent.com/38475165/78151255-b280d680-7438-11ea-9219-2f5d81b3a559.png)

**Comment by Nirjonadda** [comment]
@LukasWieditz I am using UI.X Pro Light 

![ScreenShot00186](https://user-images.githubusercontent.com/6899856/78152970-d6ddb280-743a-11ea-89e9-714c7314bb6e.png)

Result:

![ScreenShot00187](https://user-images.githubusercontent.com/6899856/78153049-f248bd80-743a-11ea-9e49-42d52f995d9f.png)

child Style: UI.X Pro Dark using Inherited value from UI.X Pro Light

![ScreenShot00188](https://user-images.githubusercontent.com/6899856/78153478-65523400-743b-11ea-9d3e-526ece7183e8.png)

Result:

![ScreenShot00189](https://user-images.githubusercontent.com/6899856/78153597-8450c600-743b-11ea-8ccb-c3f9890bfea6.png)

**Comment by LukasWieditz** [comment]
I went through all our themes and I'm not able to reproduce this issue on any of them. It looks like you've made a couple of changes to the theme. Can you try a stock version?

![mSAX8G2](https://user-images.githubusercontent.com/38475165/78157778-b4e72e80-7440-11ea-9d5f-6e8b425c9bb6.png)

**Comment by Nirjonadda** [comment]
No, I have not done any manually template edit. You are using style default nude icon? I think You are not able to reproduce this issue because you are not set Node icons. Please do setup Node icons for forum then check Sub Nodes for this forum with Inherited value.
